jenkinsapi_tests/unittests/test_build.py

```diff
@@ -215,3 +216,11 @@ def fake_get_data(cls, tree=None, params=None):
             build.get_env_vars()
     assert '' == str(excinfo.value)
     assert len(record) == 0
+
+
+def test_toggle_keep_build_forever(monkeypatch, build):
```

Comment:
Please put that test into systests - this is to ensure that it will work on actual Jenkins.